Core/iTasks.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * @file
```

Comment:
leaving these in the profile, when the profile gets substituted on an env like sites, makes them unavailable for later use, such as with the task hopper.